### DISCUSSION TITLE
Resizing Columns triggers Column Sorting

### DISCUSSION THREAD

**Discussion by eslachance** [TanStack/table#2058] [General] [answered]
**Describe the bug (required)**
When using both useResizeColumns and useSortBy , resizing the column triggers the column sorting feature. This is using all default props other than styled-components styling.

**Provide an example via Codesandbox! (required)**
Reproduced: [react-table useResizeColuns useSortBy](https://codesandbox.io/s/youthful-fast-utvo6)

**Steps To Reproduce (required)**

1. Open codesandbox
2. Resize a column

**Expected behavior**
Event bubbling is disabled, so resizing a column does not register the event in the div behind which registers the click for sorting.

**Desktop (please complete the following information):**

- OS: Ubuntu 18.04, Windows 10
- Browsers: Chromium/Chrome 80.0.3987.149, Firefox 71.0 & 74.0
- Versions: react-table "7.0.0-rc.7", "7.0.0"

**Comment by Mark-Partola** [comment]
The same

**Comment by gargroh** [comment, marked as answer]
Check https://spectrum.chat/react-table/general/i-combined-the-basic-examples-to-enable-column-sorting-and-resizing~45d0177e-36e1-47fa-9464-76f84a142469

**Comment by tannerlinsley** [comment]
> Yep, you just have to put the sortByToggleProps on an element that doesn't wrap the resizer. You can put it on a span that wraps the header text or even a div... whatever floats your boat!

**Comment by eslachance** [comment]
That was indeed the answer, thank you very much!

**Comment by dovid-rosenberg** [comment]
Thank you @tannerlinsley!

**Comment by fabricioviapiana** [comment]
Guys, how do you fix this using the v8? There's no sortByToggleProps

**Reply by webdevsyd** [reply]
same issue on my end. there is no example as a guide on how to fix this in v8 @tannerlinsley  Maybe you can have a example.
